### DISCUSSION TITLE
synthea resource reference modification

### DISCUSSION THREAD

**Discussion by sibincs0711** [synthetichealth/synthea#1369] [Q&A]
Hey , thank you for reading this.
I have been using synthea for a few days and I have noticed that synthea generates resources with references like this
"subject": {
                    "reference": "urn:uuid:51e625db-6ba0-22f2-a009-610850cd8164"
                }
which makes it difficult to fetch using observation?subject=patient id

how can i change the subject to 
"subject": {
                    "reference": "Patient/51e625db-6ba0-22f2-a009-610850cd8164"
                }

**Comment by jawalonoski** [comment]
The bundles are meant to be transaction bundles, so when you load them into a FHIR server the references are rewritten by the server during the transaction processing. In effect, transforming the references as you indicate.

However, if that is insufficient for your needs, you can force the exporter to use the format you are requesting if you move to FHIR Bulk Data, using the `exporter.fhir.bulk_data=true` property. Of course, the output will no longer be bundles.

**Reply by sibincs0711** [reply]
i have tried that, but as you said, it is not bundle. Is there any way i can configure synthea to create references like that in bundle??

**Reply by jawalonoski** [reply]
No.
